pages/expense.js

```diff
@@ -212,7 +216,7 @@ class ExpensePage extends React.Component {
 
   componentDidMount() {
     const { router, data, legacyExpenseId } = this.props;
-    const account = data?.account;
```

Comment:
I don't understand the reason for this change

Reply:
Sorry, this one is a mistake on my part. I've reverted this part.